### ISSUE TITLE
remove node-webkit embedded in source

### ISSUE THREAD

**Issue by vbfischer** [WilliamMayor/firestorm#4]
This can be automated using node-webkit-builder, and keeps the dependency for this external. If against this, that's cool. Just let me know.

**Comment by vbfischer** [comment]
Other stuff:

- [ ] update build process to use gulp/grunt
- [ ] Fix jshint issues.
- [ ] Externalize other "vendor" dependencies
- [ ] modularize the files instead of just including in the index.html. This is a little tricky apparently with node-webkit so is low priority...

**Comment by WilliamMayor** [comment]
I'm all for removing these kinds of external dependencies. Presumably this would give us the added benefit of being able to build for different platforms (i.e. Windows)?

**Comment by vbfischer** [comment]
Definitely. I created a branch with my changes so far. Think I came across the issue with it crashing will try to fix today 

—
